javascript-data-model/order-history.js

```diff
@@ -0,0 +1,77 @@
+var orderHistory = [
+  {
+    orderNumber: '114-3941689-8772232',
+    orderDate: 'August 4, 2020',
+    itemName: 'JavaScript for impatient programmers',
```

Comment:
so the thing you want to add to this data model is a property `items` which will contain an array of objects in which each object contains an individual items data: 
```javascript
var orderHis = [
  {
    orderedDate: '2020-08-04',
    totalPrice: 34,
    shipTo: 'JS Masher',
    orderNum: '114-3941689-8772232',
    deliveredDate: '2020-08-08',
    orderedDate: '2-15-21',
    items:[{
      type: "book",
      title: "A New Way to Learn"
    },
    {
      type: "electronics",
      title: "Blu Ray Player"
    },
    {
      type: "book",
      title: "How to forget"
    },
    {
      type: "book",
      title: "how to remember"
    }
  ]
  }
]
```

Reply:
Thanks for the review and comments, codes updated.
![image](https://user-images.githubusercontent.com/76827487/113452570-1a099300-93b9-11eb-9f64-379a1444abd0.png)
![image](https://user-images.githubusercontent.com/76827487/113452591-25f55500-93b9-11eb-8e0f-a1cce689f64b.png)